### PR TITLE
Playbooks for simplifying create checks

### PR DESCRIPTION
**Pull request by rndmh3ro**
Continuation of #52.

What I did additionally:

- [remove the role-dependency to simplify the creation](https://github.com/T-Systems-MMS/ansible-collection-icinga/commit/5d55e552e57618f77061751354ed245a4fbec21e)
  - the playbooks now do not call the [icinga-role](https://github.com/T-Systems-MMS/ansible-collection-icinga-director/tree/master/roles/ansible_icinga) but the tasks directly. This simplifies the code and execution, as using the role for these simple tasks is overly complex.
- [add integration tests](https://github.com/T-Systems-MMS/ansible-collection-icinga/commit/0b8e8f70781b7227a6450c84941e8f6d404e2a93)
  - the integration-tests run in GHA and only test the new playbooks. The existing roles still get tested by molecule
- [add examples to use playbooks](https://github.com/T-Systems-MMS/ansible-collection-icinga/commit/fa8630078fc72f35947dded650f1c2c732cc7830)
- [rename playbooks to remove dashes, they're not allowed.](https://github.com/T-Systems-MMS/ansible-collection-icinga/commit/37195117f51f1334741e5eca02ebbf1db786c21a)
- [put domon-stuff used by other checks in separate play](https://github.com/T-Systems-MMS/ansible-collection-icinga/commit/30c865f2ac47cea81903a6eb22281447ae1256f6)

### CODE REVIEW

File: playbooks/check_azure_oauth_token.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            skip_key: true
+            value: $scope$
+    - name: Create service template for Azure oauth token
+      telekom_mms.icinga_director.icinga_service_template:
+        state: present
+        object_name: "azure_oauth_token"
+        check_command: "azure_oauth_token"
+        check_interval: 50m
+
+# EXAMPLES
```
Comment (schurzi) [inline playbooks/check_azure_oauth_token.yml:44]: should this be somewhere in the docs/examples?

Comment (rndmh3ro) [reply playbooks/check_azure_oauth_token.yml:44]: I'm more in favor of removing them entirely. They're not examples on how to use this collection but rather what to do with the created contents..

Comment (rndmh3ro) [reply playbooks/check_azure_oauth_token.yml]: done.

**Comment by xFuture603** [comment]
As discussed with @rndmh3ro we will add a extra repository for our check scripts (f.e check_gitlab_scheduler.py) and then we will link this repository here as a dependency for various playbooks. This is a follow up task so for now the PR looks good to me.

**Review by xFuture603 — APPROVED** [review]

**Pull request merged**